web/src/beta/lib/core/Map/Layers/utils.ts

```diff
@@ -5,11 +5,10 @@ export const deepAssign = <O extends Record<string, any>>(obj: O, src: O) => {
     Object.entries({ ...src, ...obj })
       .map(([k, v]): [string, any] | undefined => {
         const srcV: unknown = src[k];
-        if (Object.hasOwn(src, k) && (srcV === undefined || srcV === null)) {
+        if (Object.hasOwn(src, k) && srcV === undefined) {
```

Comment:
When override with explicit value `null`, i think it's better to set value to `null` then ignore it. 
With this change we can clear some property value, otherwise if the layer has some property when adding, we can not clear it by override.

Reply:
Can you add test for this case?:pray:

Reply:
Sorry i realize we don't use `null` as appearance property values. So i revert this part too. When override values to reset or clear i think it's better to use empty string or empty array base on the tyep definiation.